app/models/course/discussion/topic.rb

```diff
@@ -7,6 +7,10 @@ class Course::Discussion::Topic < ActiveRecord::Base
 
   accepts_nested_attributes_for :posts
 
+  def to_partial_path
+    'course/discussion/topic'.freeze
```

Comment:
Maybe we should stick with 2.3 ? it does not look very consistent since we have frozen string literals.